shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/ApiServiceImpl.java

```diff
@@ -113,6 +113,7 @@ private String update(final ApiDTO apiDTO) {
                     .id(UUIDUtils.getInstance().generateShortUuid())
                     .apiId(apiDO.getId())
                     .tagId(tagId)
+                    .dateCreated(currentTime)
```

Comment:
Please fix the SQL statement, changing the create time in the update interface is incorrect.

Reply:
ok

Reply:
> Please fix the SQL statement, changing the create time in the update interface is incorrect.

```
  tagRelationMapper.deleteByApiId(apiDO.getId());
  tagRelationMapper.batchInsert(tags);
```
hi, but due the tag relation always be deleted and then insert new tag relation when update this api, the createdTime always is currentTime.

Reply:
oh, I saw that.